diff --git a/definitions/Imperium - Space Marines.json b/definitions/Imperium - Space Marines.json
--- a/definitions/Imperium - Space Marines.json
+++ b/definitions/Imperium - Space Marines.json
@@ -76519,7 +76519,7 @@
         "id": "9c0c-18cf-0037-d2a2",
         "hidden": false,
         "import": true,
-        "type": "upgrade",
+        "type": "unit",
         "categoryLinks": [
           {"name": "Mounted", "id": "4a47-5a17-0e89-f1f3", "hidden": false, "primary": true, "targetId": "14a0-40c9-2748-ae6e"},
           {"name": "Vehicle", "id": "e8fc-6985-f33d-eebe", "hidden": false, "primary": false, "targetId": "dbd4-63-af05-998"},
