diff --git a/definitions/Imperium - Space Marines.json b/definitions/Imperium - Space Marines.json
--- a/definitions/Imperium - Space Marines.json
+++ b/definitions/Imperium - Space Marines.json
@@ -78294,10 +78294,10 @@
               }
             ]
           },
-          {"name": "New Entry", "id": "e5e4-56a3-ca06-fecf", "hidden": false, "import": true, "type": "upgrade"},
-          {"name": "New Entry", "id": "cd09-8410-f1a0-c9af", "hidden": false, "import": true, "type": "upgrade"},
-          {"name": "New Entry", "id": "354b-c20a-541c-9420", "hidden": false, "import": true, "type": "upgrade"},
-          {"name": "New Entry", "id": "47df-bcbf-0a67-6a99", "hidden": false, "import": true, "type": "upgrade"}
+          {"name": "New Entry", "id": "e5e4-56a3-ca06-fecf", "hidden": true, "import": true, "type": "upgrade"},
+          {"name": "New Entry", "id": "cd09-8410-f1a0-c9af", "hidden": true, "import": true, "type": "upgrade"},
+          {"name": "New Entry", "id": "354b-c20a-541c-9420", "hidden": true, "import": true, "type": "upgrade"},
+          {"name": "New Entry", "id": "47df-bcbf-0a67-6a99", "hidden": true, "import": true, "type": "upgrade"}
         ]
       },
       {
